### ISSUE TITLE
Hand icon lags or hangs when started touching letter How to invalidate/redraw it when tracing it completed

### ISSUE THREAD

**Issue by Iamthemax** [farimarwat/android-word-tracing-sdk#1]
1. Hand icon lags or hangs when started touching letter.
2.  How to invalidate/redraw it when tracing it completed
3. How when letter size is increased

**Comment by farimarwat** [comment]
Yes these are the issues already noted and will be improved soon.

**Comment by online-offl** [comment]
would love to hear about the status, will try to fix things and bring up this cute app better
